### PR TITLE
BookCopy search forms & views

### PR DESCRIPTION
**Pull request by sdreher**
Backing up a bit and focusing on the search for book copies. This PR simply returns the number of book copies that are found given a set of criteria. Removing some experimental code that attempted to return the search total and the paths.

Note: this also updates the solr search schema, so please coordinate with me before merging.

### CODE REVIEW

File: footprints/templates/clientside/search_template.html [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
     <div class="widget-pane">
-        <div class="widget-pane-content">
+        <div class="widget-pane-content" v-if="criteria">
+            <div v-if="total">
+                Book Copies: {{total}}
+            </div>
+            <form>
+                <select v-model="criteria.work">
+                    <option>-----</option>
+                    <option value="67">Akedet</option>
```
Comment (nbuonin) [inline footprints/templates/clientside/search_template.html]: @sdreher was this value meant to be inlined?

File: media/js/app/components/searchvue.js [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
+        props: ['total'],
         template: '#search-template',
         data: function() {
             return {
+                criteria: {}
             };
         },
+        methods: {
+            updateCriteria: function() {
+                console.log('updateCriteria');
```
Comment (nbuonin) [inline media/js/app/components/searchvue.js]: @sdreher noting the console.log

File: footprints/templates/clientside/search_template.html (hunk 1 above)
Comment (sdreher) [reply footprints/templates/clientside/search_template.html]: a. Good catch. b. Yeah, this is okay to go to production. This is simply for some initial test purposes.

File: media/js/app/components/searchvue.js (hunk 2 above)
Comment (sdreher) [reply media/js/app/components/searchvue.js]: Resolved, ty.

**Comment by sdreher** [comment]
Solr schema has been updated. This can be pulled after it is reviewed.

**Pull request merged**